endpoints/petrol/index.js

```diff
@@ -0,0 +1,38 @@
+import request from 'request'
+import h from 'apis-helpers'
+import app from '../../server'
+
+function queryData(callback) {
+  const url = 'https://raw.githubusercontent.com/gasvaktin/gasvaktin/master/vaktin/gas.min.json'
```

Comment:
This is somewhat new for us I think and I wondering if we want to do stuff like this. Like should we keep the scraping in our project so that there aren't possible backdoors into the system?

@kristjanmik @MiniGod @benediktvaldez What do you guys think?